ccpp/CCPP_driver.F90

```diff
@@ -167,8 +169,18 @@ subroutine CCPP_step (step, nblks, ierr)
       endif
 
       !--- determine if physics diagnostics buckets need to be cleared
+      if (GFS_control%iau_offset > 0) then
+        kdt_iau = nint(GFS_control%iau_offset*3600./GFS_control%dtp)
+        if (GFS_control%kdt-1 == kdt_iau) then
+          iauwindow_center = .true.
+          if( GFS_control%me == 0)print *,'in ccpp step vary, iauwindow_center=',iauwindow_center,&
+            'kdt=',GFS_control%kdt,'dtp=',GFS_control%dtp,'iau_offset=',GFS_control%iau_offset
+        else
+          iauwindow_center = .false.
+        endif
+      endif
       if ((mod(GFS_control%kdt-1,GFS_control%nszero)) == 0) then
-        call GFS_Intdiag%phys_zero(GFS_control)
+        call GFS_Intdiag%phys_zero(GFS_control, iauwindow_center=iauwindow_center)
```

Comment:
What should be the value of `iauwindow_center` if iau_offset <= 0? It seems it is uninitialized in that case.

Reply:
it should be zero. Updated.

Reply:
kdt_iau = nint(GFS_control%iau_offset*3600./GFS_control%dtp) needs to be divided by two in order to zero out the totprcp in the beginning of the forecast. Based the experiment with the half of current kdt_iau, the totprcp was zeroed out after the first step. But the totprcp is still large in sfcf000 file.